Build/AzureDevOps/azure-pipelines.yml

```diff
@@ -14,43 +14,65 @@ jobs:
         Win32-Release:
           buildPlatform: Win32
           buildConfiguration: Release
+          vcpkgTriplet: x86-windows
         Win32-Debug:
           buildPlatform: Win32
           buildConfiguration: Debug
+          vcpkgTriplet: x86-windows
         x64-Release:
           buildPlatform: x64
           buildConfiguration: Release
+          vcpkgTriplet: x64-windows
         x64-Debug:
           buildPlatform: x64
           buildConfiguration: Debug
+          vcpkgTriplet: x64-windows
         ARM-Release:
           buildPlatform: ARM
           buildConfiguration: Release
+          vcpkgTriplet: arm-windows
         ARM-Debug:
           buildPlatform: ARM
           buildConfiguration: Debug
+          vcpkgTriplet: arm-windows
         ARM64-Release:
           buildPlatform: ARM64
           buildConfiguration: Release
+          vcpkgTriplet: arm64-windows
         ARM64-Debug:
           buildPlatform: ARM64
           buildConfiguration: Debug
+          vcpkgTriplet: arm64-windows
 
     workspace:
       clean: all
 
+    variables:
+      vcpkgRoot: "C:/vcpkg/scripts/buildsystems/vcpkg.cmake"
+
     steps:
+    - powershell: vcpkg install --triplet "$(vcpkgTriplet)" rapidjson gtest
```

Comment:
Why vcpkg? Is it better in some way? Also, is this replacing the existing NuGet way? I'm a bit confused by this.

Reply:
It doesn't replace NuGet.

In short, I want to use RapidJSON and glTF-SDK using VcPkg.  
Currently VcPkg supports RapidJSON and and Google.Test, So I thought It will be cool if glTF-SDK can be added to the package manager. By doing so users can install/use both library in same manner. That is, `find_package` with `CMAKE_TOOLCHAIN_FILE`, instead of `glTF-SDK_DIR`.

Reply:
I see. I don't know vcpkg at all, so I'm not sure if this is right, but it seems like it works at least in the CI. Does this mean that we don't run CI with the old way anymore for Windows?

Reply:
Of course, Windows users can still build without VcPkg. I will recover build jobs without VcPkg so the CI can check both ways. Thanks for the point!

Reply:
Ok, if this turns out to take a lot longer to perform the CI, it's probably okay to only test one path.